cirq-superstaq/cirq_superstaq/service.py

```diff
@@ -116,6 +116,7 @@ def __init__(
         api_version: str = gss.API_VERSION,
         max_retry_seconds: int = 3600,
         verbose: bool = False,
+        **kwargs: Any,
```

Comment:
nit (but only if it passes type check)
```suggestion
        **kwargs: object,
```